### PR TITLE
make ecr terraform scripts

### PR DESCRIPTION
**Pull request by jiuliangut**
- Created terraform script to provision each of the 3 ECRs:
- `dashboard_ecr.tf`
- `data_backup_ecr.tf`
- `etl_ecr.tf`
- Included a lifecycle policy within each script which keeps only the latest 30 images pushed to the ECR. (Likely won't affect our project's scope but could be a good consideration for managing storage costs in future)
- Updated ReadMe with basic info on terraform config so far

### CODE REVIEW

**Review by bentrzcinski — COMMENTED** [review]
Great job these look great! And love the lifecycle policy addition. Could you update the README to reflect these additions?

**Review by anitamegarry — DISMISSED** [review]
Looks good to me! Good idea including a lifestyle policy.

File: README.md [hunk 1]
```diff
@@ -7,4 +7,16 @@
...
+### Terraform Configuration:
+The `terraform` folder contains all files for terraform infrastructure:
+- `provider.tf` - defines the AWS region.
+- `s3_bucket.tf` - provisioning the AWS S3 bucket for long-term data storage (data older than 24hrs).
+  
+One script for each of the 3 ECRs utilised in this project:
+- `dashboard_ecr.tf`
+- `etl_ecr.tf`
+- `data_backup_ecr.tf`
+- Each ECR has a lifecycle policy which keeps only the 30 most recently pushed images.
```
Comment (anitamegarry) [inline README.md:22]: Nice concise explanations!

**Review by anitamegarry — DISMISSED** [review]
Looks good

File: README.md (hunk 1 above)
Comment (bentrzcinski) [reply README.md:22]: Could maybe embellish a little more on what each of the ECRs do? 'dashboard_ecr.tf' is a little vague without explaining that its for a data dashboard where users can see daily trends.

Comment (jiuliangut) [reply README.md:22]: @bentrzcinski good shout, will add descriptions for each of those

**Review by bentrzcinski — APPROVED** [review]

**Pull request merged**